Tests/Segment-Tests/JSON_Tests.swift

```diff
@@ -130,4 +130,51 @@ class JSONTests: XCTestCase {
         
         XCTAssertTrue(fetchedTraits?.email == "test@test.com")
     }
+    
+    func testKeyMapping() {
+        let keys = ["Key1": "AKey1", "Key2": "AKey2"]
+        let dict: [String: Any] = ["Key1": 1, "Key2": 2, "Key3": 3, "Key4": ["Key1": 1]]
+        
+        let json = try! JSON(dict)
+        
+        let output = try! json.mapKeys(keys).dictionaryValue
+        
+        XCTAssertTrue(output!["AKey1"] as! Int == 1)
+        XCTAssertTrue(output!["AKey2"] as! Int == 2)
+        XCTAssertTrue(output!["Key3"] as! Int == 3)
+        
+        let subDict = output!["Key4"] as! [String: Any]
+        XCTAssertTrue(subDict["AKey1"] as! Int == 1)
+    }
+    
+    func testKeyMappingWithValueTransform() {
+        let keys = ["Key1": "AKey1", "Key2": "AKey2"]
+        let dict: [String: Any] = ["Key1": 1, "Key2": 2, "Key3": 3, "Key4": ["Key1": 1], "Key5": [1, 2, ["Key1": 1]]]
+        
+        let json = try! JSON(dict)
+        
+        let output = try! json.mapKeys(keys, valueTransform: { key, value in
+            var newValue = value
+            if let v = newValue as? Int {
+                if v == 1 {
+                    newValue = 11
+                }
+            }
+            print("value = \(value.self)")
+            return newValue
+        }).dictionaryValue
+        
+        XCTAssertTrue(output!["AKey1"] as! Int == 11)
```

Comment:
Can we change these to as? so we fail the unit test instead of crash the unit tests?

Reply:
We could I guess, though that doesn't really bother me.  We'd need to go hit a bunch of other tests as well as that's my MO for unit tests.  FORCE UNWRAP ALL THE THINGS!